tools/azsdk-cli/Azure.Sdk.Tools.Cli.Tests/Helpers/LabelHelperTests.cs

```diff
@@ -0,0 +1,58 @@
+using Azure.Sdk.Tools.Cli.Helpers;
+using Azure.Sdk.Tools.Cli.Tests.TestHelpers;
+using Azure.Sdk.Tools.Cli.Configuration;
+
+
+namespace Azure.Sdk.Tools.Cli.Tests.Helpers;
+
+internal class LabelHelperTests
+{
+    private ILabelHelper labelHelper;
+
+    [SetUp]
+    public void Setup()
+    {
+        labelHelper = new LabelHelper(new TestLogger<LabelHelper>());
+    }
+
+    [Test]
+    [TestCase("TestService,Description,e99695\nAnotherService,Description2,e99695", "TestService", LabelHelper.ServiceLabelStatus.Exists)]
+    [TestCase("TestService,Description,e99695\r\nAnotherService,Description2,e99695\nThirdService,Description3,e99695", "AnotherService", LabelHelper.ServiceLabelStatus.Exists)]
+    [TestCase("TestService,Description,e99695\nAnotherService,Description2,e99695\n\r", "TestService", LabelHelper.ServiceLabelStatus.Exists)]
+    [TestCase("TestService,Description,e99695\nAnotherService,Description2,e99695", "NonExistentService", LabelHelper.ServiceLabelStatus.DoesNotExist)]
+    [TestCase("TestService,Description,123456\nAnotherService,Description2,e99695", "TestService", LabelHelper.ServiceLabelStatus.NotAServiceLabel)]
+    [TestCase("Service - TestService,Description with commas and stuff,e99695\nAnotherService,Description2,e99695", "Service - TestService", LabelHelper.ServiceLabelStatus.Exists)]
+    [TestCase("Service - TestService,Description with commas and stuff,e99695\nAnotherService,Description2,e99695", "AnotherService", LabelHelper.ServiceLabelStatus.Exists)]
+    public void TestCheckServiceLabel_DataDriven(string csvContent, string serviceLabel, LabelHelper.ServiceLabelStatus expected)
```

Comment:
These look great now! Easier to see at a glance all the different cases we're testing.

I don't think the `_DataDriven` name in the test is necessary.